### ISSUE TITLE
Sharp curves seem to flip the input path

### ISSUE THREAD

**Issue by ctrlcctrlv** [mfeq/Qstroke#4]
I assumed that I just didn't have enough points on the input path, but adding more points to `22.glif` doesn't substantially improve the output.

```
Qstroke --out Untitled2.ufo/glyphs/22.glif --path Untitled1.ufo/glyphs/22.glif --pattern arrow.ufo/glyphs/arrow.glif -m repeated --sx 0.1 --sy 0.1 -s 2
```

[4.zip](https://github.com/mfeq/Qstroke/files/5792024/4.zip)
![image](https://user-images.githubusercontent.com/838783/104111452-d9194b00-52af-11eb-9e26-aaee29bc001a.png)

**Comment by MatthewBlanchard** [comment]
![fixed_curvature](https://user-images.githubusercontent.com/310356/104128885-090c3100-5338-11eb-8078-954fa35b7dc2.PNG)

Issue fixed in coming commit.

**Comment by ctrlcctrlv** [comment]
Confirmed solved by 1518cfa3479dbbbe39fc5c340d027a9c2a9f0837 , 5dfac7fc2e5e920d51b021790fc898c455cb4a0d

![image](https://user-images.githubusercontent.com/838783/104132949-8d69ae80-534e-11eb-811f-afd18e5fe405.png)
